charts/px-backup/templates/px-central-ui/pxcentral-ui.yaml

```diff
@@ -245,7 +246,11 @@ metadata:
   namespace: {{ .Release.Namespace }}
 type: Opaque
 stringData:
-  DB_PASSWORD: "Password1"
+  {{- if eq $operatorToChartUpgrade true }}
+  DB_PASSWORD: singapore
+  {{- else }}
+  DB_PASSWORD: Password1
```

Comment:
just for my knowledge, what is the background behind the password change? And why `singapore` specifically?

Reply:
This password is used by UI backend MySQL DB, Binu has hardcoded it in the previous release. so to support operator to chart upgrade we need to make it configurable and add/paas old password value.